x/superfluid/types/codec.go

```diff
@@ -14,6 +14,7 @@ func RegisterCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&MsgSuperfluidUndelegate{}, "osmosis/superfluid-undelegate", nil)
 	cdc.RegisterConcrete(&MsgLockAndSuperfluidDelegate{}, "osmosis/lock-and-superfluid-delegate", nil)
 	cdc.RegisterConcrete(&MsgSuperfluidUnbondLock{}, "osmosis/superfluid-unbond-lock", nil)
+	cdc.RegisterConcrete(&MsgSuperfluidUndelegateAndUnbondLock{}, "osmosis/superfluid-undelegate-and-unbond-lock", nil)
```

Comment:
yup any suggestion from the core team?
How about `MsgSuperfluidUnboundLockPartial`?

Reply:
I think `MsgSuperfluidUnboundLockPartial` can cause confusion with `MsgSuperfluidUnbondLock` since its doing much more than `MsgSuperfluidUnbondLock`. I suggest using shorter terms for superfluid, 
`osmosis/superfluid-undelegate-and-unbond-lock` -> `osmosis/sf-undelegate-and-unbond-lock`